app/components/publications/batch-documents-publication-row.hbs

```diff
@@ -49,3 +49,10 @@
     </WebComponents::AuButtonToolbar>
   </td>
 </tr>
+
+{{#if this.isOpenNewPublicationModal}}
+  <Publications::NewPublicationModal
+    @case={{this.case}}
+    @onSave={{this.saveNewPublication}}
+    @onCancel={{this.cancelNewPublication}} />
+{{/if}}
```

Comment:
### NewPublicationModal
Niet akkoord: Dit zorgt ervoor dat de NewPublicationModal getoond wordt binnen de css van de met een css-class als full-screen gemarkeerde BatchDocumentsPublicationModal. Hierdoor is de NewPublicationModal ook fullscreen. Ik heb er ook moeite mee dat er dan (code-matig) een NewPublicationModal bestaat voor elke row, maar dat kan aan mij liggen.

Ik zie niet meteen een properdere workaround. Daarom zou ik best werken met een context object. Als ge de document-grouping wilt centraal houden zou ik hiervoor een static functie toevoegen aan Piece.

Een andere mogelijkheid misschien: Ik heb gezien dat de powerselect de Ember-Wormhole-addon gebruikt.

Ik vind het ook jammer dat de fade-out animatie bij sluiten van de modal niet in de applicatie zit.

### selected property
Met de selected computed property zorgde ik ervoor dat de juiste publicatieflow in de optionslijst van de selectbox geselecteerd was. Dat is iets vreemd met EmberPowerSelect: het model meegegeven met @onchange(selected) en het model in de lijst van @options zijn in sommige gevallen niet dezelfde instanties. Ik ben er nog niet uit wanneer dit zo is, maar in deze modal was dit zo.

Reply:
Over de positionering van de new publication modal heb je gelijk. Ik heb deze terug verhuisd naar de batch-documents-publication-modal. Met een `did-update` modifier wordt er nu in de row-component gezorgd dat de selected option aangepast wordt wanneer de publication flow van het document wijzigt.